js/src/ui/Translate/translate.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+import { Translate as I18NTranslate } from 'react-i18nify';
+
+export default class Translate extends Component {
```

Comment:
- Shrinkwrap - https://github.com/ethcore/parity/issues/3520
- `react-i18nify` is not set in stone, and with the component issues, may make sense to use the lower-level I18N.t/I18N.l versions instead. So makes 100% sense not using the original.